testing/test_saucelabs.py

```diff
@@ -77,6 +78,83 @@ def test_invalid_credentials_file(failure, monkeypatch, tmpdir):
     assert any(message in out for message in messages)
 
 
+def test_no_sauce_options(monkeypatch, testdir):
+    username = 'foo'
+    access_key = 'bar'
+    monkeypatch.setenv('SAUCELABS_USERNAME', username)
+    monkeypatch.setenv('SAUCELABS_API_KEY', access_key)
+
+    capabilities = {'browserName': 'chrome'}
+    variables = testdir.makefile('.json', '{{"capabilities": {}}}'.format(
+        json.dumps(capabilities)))
+
+    file_test = testdir.makepyfile("""
+        import pytest
+        @pytest.mark.nondestructive
+        def test_sauce_capabilities(driver_kwargs):
+            assert driver_kwargs['desired_capabilities']['username'] == '{}'
+            assert driver_kwargs['desired_capabilities']['accessKey'] == '{}'
```

Comment:
The purpose of this test is to check that `sauce:options` is not present, so I don't think we need to check the capabilities here.

Reply:
Dammit, I should've followed my instincts. Will move them out to their own test. 👍 

Will add a section to the release-notes as well. :)